content/en/docs/reference/glossary/infrastructure-resource.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Resource (infrastructure)
+id: infrastructure-resource
+date: 2025-02-09
+short_description: >
+  A defined amount of infrastructure available for consumption (CPU, memory, etc).
+
+aka:
+tags:
+- fundamental
+---
+Capabilities provided by {{< glossary_tooltip text="nodes" term_id="node" >}} (CPU, memory, GPUs, etc), and made available for consumption by processes or users of those nodes.
+<br>
```

Comment:
We can avoid HTML here:
```suggestion
Capabilities provided to one or more {{< glossary_tooltip text="nodes" term_id="node" >}} (CPU, memory, GPUs, etc), and made available for consumption by
{{< glossary_tooltip text="Pods" term_id="pod" >}} running on those nodes.

```

Reply:
Note: the rendered text doesn't have a line break